### PR TITLE
sl-48: * Login functionality

### PR DESCRIPTION
**Pull request by Alexandr-mas**

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -48,5 +49,8 @@
   "simple-git-hooks": {
     "pre-commit": "npx lint-staged",
     "commit-msg": "npx commitlint --edit $1"
+  },
+  "dependencies": {
+    "bcrypt": "5.1.0"
```
Comment (andanf-e) [inline package.json]: `bcrypt` should be installed on backend

File: backend/src/packages/users/user.entity.ts [hunk 2]
```diff
@@ -87,6 +87,8 @@ class UserEntity implements IEntity {
   public toObject(): {
     id: number;
     email: string;
+    passwordHash: string;
+    passwordSalt: string;
```
Comment (andanf-e) [inline backend/src/packages/users/user.entity.ts]: Create separate entity method for this payload
```suggestion

```

**Review by andanf-e — CHANGES_REQUESTED** [review]
All `bcrypt` methods should be used in separate encryptionService, and it should be used all over the places where you need

**Comment by Alexandr-mas** [comment]
image
If the user is not found, then the network shows the error status and writes the message User Not Exist, the same for the password, writes Password Invalid
image
And if everything is fine, then it sends status 200

**Review by what1s1ove — APPROVED** [review]

**Review by andanf-e — APPROVED** [review]

**Pull request merged**